src/main/java/kitchenpos/ui/ProductRestController.java

```diff
@@ -20,16 +23,16 @@ public ProductRestController(final ProductService productService) {
     }
 
     @PostMapping("/api/products")
-    public ResponseEntity<Product> create(@RequestBody final Product product) {
-        final Product created = productService.create(product);
-        final URI uri = URI.create("/api/products/" + created.getId());
+    public ResponseEntity<ProductResponse> create(@RequestBody final ProductRequest productRequest) {
+        final ProductResponse response = productService.create(productRequest);
+        final URI uri = URI.create("/api/products/" + response.getId());
         return ResponseEntity.created(uri)
-                .body(created)
+                .body(response)
                 ;
```

Comment:
다른 부분들도 확인 부탁드리겠습니다 🙏 

```suggestion
                .body(response);
```